news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -159,49 +162,34 @@ protected function parseArticle($objArticle, $blnAddArchive=false, $strClass='',
 		$objTemplate->addImage = false;
 		$objTemplate->addBefore = false;
 
-		// Add an image
-		if ($objArticle->addImage && $objArticle->singleSRC)
+		// Add image
+		if ($objArticle->addImage && null !== ($figureBuilder = $this->getFigureBuilderIfResourceExists($objArticle->singleSRC)))
 		{
-			$objModel = FilesModel::findByUuid($objArticle->singleSRC);
-
-			if ($objModel !== null && is_file(System::getContainer()->getParameter('kernel.project_dir') . '/' . $objModel->path))
+			// If the external link is opened in a new window, open the image link in a new window, too (see #210)
+			if ('external' === $objTemplate->source && $objTemplate->target)
 			{
-				// Do not override the field now that we have a model registry (see #6303)
-				$arrArticle = $objArticle->row();
-
-				// Override the default image size
-				if ($this->imgSize)
-				{
-					$size = StringUtil::deserialize($this->imgSize);
-
-					if ($size[0] > 0 || $size[1] > 0 || is_numeric($size[2]) || ($size[2][0] ?? null) === '_')
-					{
-						$arrArticle['size'] = $this->imgSize;
-					}
-				}
-
-				$arrArticle['singleSRC'] = $objModel->path;
-				$this->addImageToTemplate($objTemplate, $arrArticle, null, null, $objModel);
+				$figureBuilder->setLinkAttribute('target', '_blank');
+			}
 
-				// Link to the news article if no image link has been defined (see #30)
-				if (!$objTemplate->fullsize && !$objTemplate->imageUrl)
-				{
-					// Unset the image title attribute
-					$picture = $objTemplate->picture;
-					unset($picture['title']);
-					$objTemplate->picture = $picture;
+			$figure = $figureBuilder
+				->setSize($objArticle->size)
```

Comment:
The `$this->imgSize` override seems to be missing here.

Reply:
Fixed in 762b0f65877d986a37bfafcfafb4ac6195ca5b2e